src/main/java/uk/gov/gchq/palisade/service/ResourceService.java

```diff
@@ -45,18 +46,18 @@
      * details on the contained files.
```

Comment:
Comment above here mentions @link Streams which doesn't appear to be correct now,

Reply:
Nice spot. I have updated this.